.gitignore

```diff
@@ -3,5 +3,7 @@
 cmake-build-debug/
 cmake-build-release/
 build/
+install/
```

Comment:
this seems outside the scope of this feature

Reply:
correct, should be removed